beacon/types/config.go

```diff
@@ -99,9 +106,14 @@ func (f Forks) SigningRoot(header Header) (common.Hash, error) {
 	return signingRoot, nil
 }
 
-func (f Forks) Len() int           { return len(f) }
-func (f Forks) Swap(i, j int)      { f[i], f[j] = f[j], f[i] }
-func (f Forks) Less(i, j int) bool { return f[i].Epoch < f[j].Epoch }
+func (f Forks) Len() int      { return len(f) }
+func (f Forks) Swap(i, j int) { f[i], f[j] = f[j], f[i] }
+func (f Forks) Less(i, j int) bool {
+	if f[i].knownIndex != f[j].knownIndex {
+		return f[i].knownIndex < f[j].knownIndex
+	}
+	return f[i].Epoch < f[j].Epoch
```

Comment:
I think it should be flipped here? The knownIndex only matters when the epoch is equal?

Reply:
Indeed, this is what I meant, fixed.